src/config/config.c

```diff
@@ -1317,7 +1317,7 @@ static void initConfig(struct config *conf)
 	conf->misc.addr2line.c = validate_stub; // Only type-based checking
 
 	conf->misc.etc_dnsmasq_d.k = "misc.etc_dnsmasq_d";
-	conf->misc.etc_dnsmasq_d.h = "Should FTL load additional dnsmasq configuration files from /etc/dnsmasq.d/?";
+	conf->misc.etc_dnsmasq_d.h = "Should FTL load additional dnsmasq configuration files from /etc/dnsmasq.d/?\n Warning: This is an advanced setting and should only be used with care. Incorrectly formatted or config files specifying options which can only be defined once can result in conflicts with the automatic configuration of Pi-hole (see "DNSMASQ_PH_CONFIG"). This will stop DNS resolution from working.\n Use this option with extra care.";
```

Comment:
`DNSMASQ_PH_CONFIG` where should one see documentation for this?

This help text emphasis `taking care` twice. I think once is enough

Reply:
`DNSMASQ_PH_CONFIG` is a code definition
![Screenshot from 2025-03-03 11-47-08](https://github.com/user-attachments/assets/d7ca84d2-76f4-4944-a1af-946cbd1a0081)
it expands to
![image](https://github.com/user-attachments/assets/c6963f30-a407-412c-9c61-2cbf14ee7775)
in the final text., see
![image](https://github.com/user-attachments/assets/0bcda360-876f-4e6a-9174-996ec5de6927)


> This help text emphasis taking care twice. I think once is enough

I did just edit this away.